bot.py

```diff
@@ -111,7 +111,7 @@
             links += "\n \n"
 
         links += "```"
-        await message.channel.send(links)
```

Comment:
you.... took out the thing that sent the message

Reply:
im confused

Reply:
`await message.channel.send(links)` is the bit of code that sends the message